pkg/sentry/server/validator/internal/common_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/stretchr/testify/assert"
+
+	sentryv1pb "github.com/dapr/dapr/pkg/proto/sentry/v1"
+)
+
+func Test_Validate(t *testing.T) {
```

Comment:
Why?

I use `_` as a separator so that when testing un-exported functions, we can maintain the case of the tested function. It also improves the readability of test run output.

i.e. `Testfoo` vs `Test_foo`.

Reply:
by convention the name starts with `TestXyz` https://pkg.go.dev/testing